fenix/app/src/main/java/org/mozilla/fenix/components/metrics/AdjustMetricsService.kt

```diff
@@ -91,6 +91,8 @@ class AdjustMetricsService(
                 if (event is Event.GrowthData && storage.shouldTrack(event)) {
                     Adjust.trackEvent(AdjustEvent(event.tokenName))
                     storage.updateSentState(event)
+                } else if (event is Event.ActivatedUserData && storage.shouldTrack(event)) {
+                    storage.updateSentState(event)
```

Comment:
I don't think this branch will get hit, because there are no concrete types that implement `ActivatedUserData`.

Reply:
I removed `ActivatedUserData`, see below comment on `Event.kt`

there's new logic to either send the event or update local data for determining if `AdjustMetricsService` should send event.